README.md

```diff
@@ -31,7 +31,7 @@
 
 ### Requirements
 
-- Sublime text 3 or above
+- Sublime text 4
```

Comment:
Technically it doesn't have a version number anymore, only build number. So could say something like:

```suggestion
- Sublime text (build 4xxx or later)
```